src/main/java/nl/tudelft/contextproject/model/EntityState.java

```diff
@@ -1,19 +0,0 @@
-package nl.tudelft.contextproject.model;
-
-/**
- * Enum representing the state of an Entity.
- */
-public enum EntityState {
-	/**
-	 * The entity is dead and should be removed from the renderer.
-	 */
-	DEAD,
-	/**
-	 * The entity is alive and should receive updates.
-	 */
-	ALIVE,
-	/**
-	 * The entity is new and should be added to the renderer.
-	 */
-	NEW 
```

Comment:
(For others also reviewing this, EntityState was moved to the entities package, and since the diff was so great, git did not see it as a MOVE, but rather as a delete + add. See below for the new EntityState).